### PR TITLE
Graphics fixes

### PR DESCRIPTION
**Pull request by bjacquez**
Ducking and stuff works for stuff.

### CODE REVIEW

**Comment by cudilara** [comment]
At last! Thank you :)

**Comment by iffig** [comment]
Its beautiful! Merge!!!!

File: src/com/csciOOD/DamageTracker.java [hunk 1]
```diff
@@ -7,7 +7,11 @@
 
 public class DamageTracker implements Tracker {
 
-    int damageTracker = 100;
+    int damageTracker = 50;
```
Comment (Ditofry) [inline src/com/csciOOD/DamageTracker.java:10]: Now our game will be more competitive  🎮

**Comment by Ditofry** [comment]
Looks good!  :shipit:

**Pull request merged**